components/Document/lib/sharedGetStaticProps.ts

```diff
@@ -84,13 +84,15 @@ export default async function sharedGetStaticProps({
       }
 
       const filter = getCommentFilterByTab(tabName);
-      commentData = await fetchCommentsAPI({
-        documentId,
-        documentType:
-          documentType === "post" ? "researchhubpost" : documentType,
-        // @ts-ignore
-        filter,
-      });
+      if (filter !== null) {
+        commentData = await fetchCommentsAPI({
```

Comment:
This might be out of sync with master now, I believe we've moved this out of the getStaticProps (was causing a bug due to too much data)

Reply:
That is correct